### DISCUSSION TITLE
Connect YetiForce to existing MySQL Databse

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14777] [How to]
Hi,
I have already a MYSQL database in cloud (with companies and contacts records).
I would connect these data to YetiForce and so using these tables.
It is possible?

Thanks

**Comment by mariuszkrzaczkowski** [comment]
config/Db.php add 
```php
public static function newDb()
{
	return [
		'dsn' => 'mysql:host=xxx.yetiforce.com;dbname=yyy;port=3306',
		'host' => 'xxx.yetiforce.com',
		'port' => 3306,
		'username' => 'yyy',
		'password' => '.....',
		'dbName' => 'yyy',
		'charset' => 'utf8',
		'attributes' => [
			\PDO::ATTR_TIMEOUT => 2,
		],
	];
}
```
```php
$db = \App\Db::getInstance('newDb');
...
```

**Reply by espe808** [reply]
@mariuszkrzaczkowski 

**Done this!**

public static function newDb()
{
	return [
		'dsn' => 'mysql:host=xxx.yetiforce.com;dbname=yyy;port=3306',
		'host' => 'xxx.yetiforce.com',
		'port' => 3306,
		'username' => 'yyy',
		'password' => '.....',
		'dbName' => 'yyy',
		'charset' => 'utf8',
		'attributes' => [
			\PDO::ATTR_TIMEOUT => 2,
		],
	];
}

-----------------


$db = \App\Db::getInstance('newDb');
...

**This one, where I have to write?
The file I mean**
**And then where I see these newDB front-end side?**

**Reply by espe808** [reply]
Solved.
